lib/engine/game/g_18_uruguay/step/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/buy_sell_par_shares'
+
+module Engine
+  module Game
+    module G18Uruguay
+      module Step
+        class BuySellParShares < Engine::Step::BuySellParShares
+          def can_buy?(entity, bundle)
+            return false if bundle.corporation == @game.rptla && @game.phase.current[:name] == '2'
+
+            super(entity, bundle)
```

Comment:
A couple of very minor points:

- As you're checking the current phase, you can just write `@game.phase.name` instead of `@game.phase.current[:name]`.
- Calling `super` with no arguments results in a call to the superclass's method of the same name, with the same arguments passed. It's more normal for this codebase to use the bare `super` call, instead of explicitly passing on the arguments.

Have you considered using a phase status for when these RPTLA shares are available? It would make this visible to players on the 'info' tab.

Reply:
Good suggestions

Updated @game.phase.name and updated phase status